test/setup.sh

```diff
@@ -15,3 +15,9 @@ for HOST in $HOSTS; do
     run_on $HOST chmod a+x $DOCKER_NS ./weave
     run_on $HOST sudo service docker restart
 done
+
+echo Setting up Docker TLS on hosts
+(cd ./tls && go get ./... && go run generate_certs.go $HOSTS)
+for HOST in $HOSTS; do
+    rsync -az -e "$SSH" ./tls/ $HOST:~/tls
+done
```

Comment:
make this part of the earlier `for` loop.

Also, I am guessing there is a reason we don't use rsync elsewhere. @squaremo might know.